test/Microsoft.ML.OnnxTransformerTest/OnnxTransformTests.cs

```diff
@@ -332,6 +332,78 @@ public void OnnxModelMultiInput()
             }
         }
 
+        [OnnxFact]
+        public void OnnxModelOutputDifferentOrder()
+        {
```

Comment:
Maybe you can refactor both tests into one, as they're very similar. And include an `[InlineData(true)` with `OnnxModelOutputDifferentOrder(bool checkSubset)`?

Reply:
I'll combine them. I dont think the `[InlineData] is needed since I'm testing the output ordering and not the `checkSubset` itself though.

Reply:
I misunderstood what you were saying first. I get it now. The issue with that is that with the subset I dont compare all the columns and with the full column set I do. I just combined the tests and moved the subset pipeline and checking below the full one.